classes/Context.php

```diff
@@ -364,16 +365,23 @@ public function updateCustomer(Customer $customer)
     /**
      * @return Translator
      */
-    public function getTranslator()
+    public function getTranslator($needComponentTranslator = false)
```

Comment:
I made it possible to tell via a parameter which translator you need, because during installation it would get the framework bundle, which is configured to init the db sql loader, but the installer calls it before the database is ready....

Reply:
I don't think this is a good idea, because

1. TranslatorComponent is meant to go away eventually
2. It exposes internal functionality
3. It's impossible tu understand if you don't know the underlying complexities 🗡 

If this is a special case just for the installer, let's just own it up:

```suggestion
    public function getTranslator($isInstaller = false)
```

or alternatively:

```suggestion
    public function getTranslator($withDatabase = true)
```

In any case, this parameter must be declared and explained in Phpdoc

Reply:
done !